static/manifest.json

```diff
@@ -18,34 +18,21 @@
   "content_scripts": [
     {
       "all_frames": true,
-      "js": [
-        "contentscript.js"
-      ],
-      "matches": [
-        "http://*/*",
-        "https://*/*"
-      ],
+      "js": ["contentscript.js"],
+      "matches": ["http://*/*", "https://*/*"],
       "run_at": "document_start"
     },
     {
-      "matches": [
-        "*://connect.trezor.io/9/popup.html"
-      ],
-      "js": [
-        "./vendor/trezor-content-script.js"
-      ]
+      "matches": ["*://connect.trezor.io/9/popup.html"],
+      "js": ["./vendor/trezor-content-script.js"]
     }
   ],
   "content_security_policy": {
-    "extension_pages": "frame-ancestors 'none'; script-src 'self'; object-src 'self'; connect-src 'self'"
+    "extension_pages": "frame-ancestors 'none'; script-src 'self' 'wasm-unsafe-eval'; object-src 'self'; connect-src 'self'"
```

Comment:
Is this a requirement from the SDK? Would it work if we don't add it?

Reply:
Removed the `wasm-unsafe-eval` option. Seems to be working fine, but I remember hitting an issue of blank popup page when I didn't have this option on. I think it's gone, because I added resolutions to `@ethereumjs/common` and `@ethereumjs/tx` which eliminates potential mismatch.